src/d3d11/d3d11_context.cpp

```diff
@@ -386,10 +386,17 @@ namespace dxvk {
     const UINT                              Values[4]) {
     D3D10DeviceLock lock = LockContext();
 
-    auto uav = static_cast<D3D11UnorderedAccessView*>(pUnorderedAccessView);
+    if (!pUnorderedAccessView)
+      return;
 
-    if (!uav)
+    Com<ID3D11UnorderedAccessView> qiUav;
+
+    if (FAILED(pUnorderedAccessView->QueryInterface(IID_PPV_ARGS(&qiUav))))
+    {
       return;
+    }
```

Comment:
formatting nit (no reason to have {} here, and we never put { on separate lines)